inc/src/Contracts/Config/LoaderInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace MyBB\Contracts\Config;
+
+interface LoaderInterface
+{
+    /**
+     * Get the specified configuration value.
+     *
+     * @param string $key
+     * @param null $default
```

Comment:
Should be `@param string|null $default A defualt value to use if the specified configuration value doesn't exist`.